etc/kayobe/seed.yml

```diff
@@ -155,23 +155,101 @@ seed_extra_containers: {}
 ###############################################################################
 # Seed node firewalld configuration.
 
+# FIXME: Replace concrete names (provision_oc) with abstract net names ({{
+# provision_oc_net_name }}).
+
 # Whether to install and enable firewalld.
-#seed_firewalld_enabled:
+seed_firewalld_enabled: true
 
 # A list of zones to create. Each item is a dict containing a 'zone' item.
-#seed_firewalld_zones:
+seed_firewalld_zones: "{{ seed_firewalld_zones_default | union(seed_firewalld_zones_extra) | unique | select }}"
+seed_firewalld_zones_extra: []
+seed_firewalld_zones_default: "{% set network_zones = [] %}{% for network in network_interfaces %}{% set _ = network_zones.append({'zone': network | net_zone}) %}{% endfor %}{{ network_zones }}"
 
 # A firewalld zone to set as the default. Default is unset, in which case the
 # default zone will not be changed.
-#seed_firewalld_default_zone:
+seed_firewalld_default_zone: drop
 
 # A list of firewall rules to apply. Each item is a dict containing arguments
 # to pass to the firewalld module. Arguments are omitted if not provided, with
 # the following exceptions:
 # - offline: true
 # - permanent: true
 # - state: enabled
-#seed_firewalld_rules:
+seed_firewalld_rules: "{{ seed_firewalld_rules_default | selectattr('enabled', 'true') | map(attribute='rules') | flatten | selectattr('network', 'in', network_interfaces) | selectattr('zone') | union(seed_firewalld_rules_extra) | unique | select }}"
+seed_firewalld_rules_extra: []
+seed_firewalld_rules_default:
+  # Common
+  - rules:
+      - service: ssh
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: dhcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: tftp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: ntp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: ntp
+        zone: "{{ switch_mgmt_net_name | net_zone }}"
+        network: "{{ switch_mgmt_net_name }}"
+        state: enabled
```

Comment:
I think this one might be environment specific

Reply:
Removed